walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -223,6 +284,34 @@ public void justEnoughRoom() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void optionalJustEnoughRoom () {
+    // Have one person, but make it so that there is just enough room at one point in the day to
+    // have the meeting. With optional Person C, there is not enough time, so disregard them. 
+    //
+    // Events  : |--A--|     |----A----|
+    // Optional:       |-C-|
```

Comment:
To show that C's event starts before A's ends

```suggestion
    // Optional:     |-C-|
```

Reply:
Oh, and probably should stretch out this diagram of C's meeting to show that it ends at the same time as A's second event starts.